### PR TITLE
Added pulsar batching and chunking sample

### PR DESCRIPTION
**Pull request by prapatil-tibco**
Added pulsar batching and chunking sample

### CODE REVIEW

File: samples/app-dev/Connectors/MessagingAndEventPubSub/PulsarSample/PulsarChunkingAndBatching/readme.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
+# Flogo Application sample for Apache Pulsar Connector
```
Comment (vinsharm-tibco) [inline samples/app-dev/Connectors/MessagingAndEventPubSub/PulsarSample/PulsarChunkingAndBatching/readme.md:1]: Please rename Heading as
Sample about Batching and Chunking feature in Apache Pulsar Connector.

File: samples/app-dev/Connectors/MessagingAndEventPubSub/PulsarSample/PulsarChunkingAndBatching/readme.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+3. On the app details page, select Import app.
+![Select import](../../../../import-screenshots/3.png)
+
+4. Browse on your machine or drag and drop the .json file for the app that you want to import.
+![Import your sample](../../../../import-screenshots/pulsar/ImportApp.png)
+
+5. Click Upload. The Import app dialog displays some general warnings and specific error in you app if any.
+![The Import app dialog](../../../../import-screenshots/pulsar/ImportDialog.png)
+
+6. You have the option to import all flows from the source app or selectively import flows.
```
Comment (vinsharm-tibco) [inline samples/app-dev/Connectors/MessagingAndEventPubSub/PulsarSample/PulsarChunkingAndBatching/readme.md:32]: Remove this point as this is not available now.

File: samples/app-dev/Connectors/MessagingAndEventPubSub/PulsarSample/PulsarChunkingAndBatching/readme.md [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+
+7. If you choose selective import, select the trigger, flow and connection. Click Next.
```
Comment (vinsharm-tibco) [inline samples/app-dev/Connectors/MessagingAndEventPubSub/PulsarSample/PulsarChunkingAndBatching/readme.md:34]: Remove this

File: samples/app-dev/Connectors/MessagingAndEventPubSub/PulsarSample/PulsarChunkingAndBatching/readme.md [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+* In TLS or JWT auth types, broker URL requires 'pulsar+ssl' in the beginning of the URL. In plain mode, only 'pulsar' is required.
+* Update the Broker URL and certificates in the connection if using TLS mode. Refer to Pulsar Security documentation to get the idea about auth types and creating certificates- https://pulsar.apache.org/docs/en/security-overview  
+For more information on connection fields, please refer to https://integration.cloud.tibco.com/docs/index.html#Subsystems/flogo-pulsar/connector-users-guide/pulsar-connection-details.html
+* Connecting to Pulsar hosted on Tibco Cloud Messaging - You can also connect to the pulsar TCM instance by configuring Pulsar Connection using tcm-config.yaml. In this case you need to use Autorization type as JWT and use the tcm_authentication_key as the JWT token. You can get the broker url and namespace from the same tcm-config.yaml.
+Please refer to - https://messaging.cloud.tibco.com/docs/getstarted/quickstart-pulsar/index.html
+
+### The Producer and Consumer Flows
```
Comment (vinsharm-tibco) [inline samples/app-dev/Connectors/MessagingAndEventPubSub/PulsarSample/PulsarChunkingAndBatching/readme.md:54]: As you are using 3 applications. Please give proper heading for all 3 app name and then description about flow.

File: samples/app-dev/Connectors/MessagingAndEventPubSub/PulsarSample/PulsarChunkingAndBatching/readme.md [hunk 5]
```diff
@@ -0,0 +1,127 @@
+# Flogo Application sample for Apache Pulsar Connector
+
+
+## Description
+
+This samples demonstrate a simple worflow about how pulsar publisher and consumer work through a flogo application using the Chunking and batching feature.
+
+The sample application has two flows: the first is 'Pulsar-producer', which publishes messages to a persistent Pulsar topic using the chunking feature and returns the payload of published messages, and the other flow has a Pulsar consumer trigger, which listens to the same topic using a Pulsar subscription and aggregates all the chunks and receives the original message.And for batching Pulsar-producer', which publishes messages in batches to a persistent Pulsar topic and returns the payload of published messages, and the other consumer app has a Pulsar consumer trigger, which listens to the same topic using a Pulsar subscription and receives the batch of messages together.
```
Comment (vinsharm-tibco) [inline samples/app-dev/Connectors/MessagingAndEventPubSub/PulsarSample/PulsarChunkingAndBatching/readme.md:8]: You can just describe more about batching and chunking here and for application and flow you can add detailed description in the Flow section.

File: samples/app-dev/Connectors/MessagingAndEventPubSub/PulsarSample/PulsarChunkingAndBatching/PulsarBatchingConsumer.json [hunk 6]
```diff
@@ -0,0 +1,236 @@
...
...EJBUUVGQUFPQ0FROEFNSUlCQ2dLQ0FRRUFycFMySG1CTndGOUJvQVR4M0U3bVpUNk5zdmhaCnBQdmtyYVo1VE1PajdUVHdvTU51TWpwTWNUZzRRc3BnckhGNHduK3lDL1VmTVdDamxWVnN4dTE3K3JXT0VmKy8KYWYySmdUUTBrdi9oWng5SWprTm1IL2YxSjdrZUorbXBEVm5nNjV5K0ZVQ2gyRjBXNG1mU2gzYXJJTGdMQ2FzUwpFK1dpSTl1dG9leG9sQzNuYWNHU1JFWFZpYmpodFQrR3dBb3dOc2lQVkN1TjBRdjczeWhDVHE0K1g1azlmTnREClp6UU5lNXB4eW1yQ056T0JxcldLZVN3aTNxUjNhd2JIRCtGaG1YaVZEN29rMnRwOW1ybmwxZkRDbEcyODhDQkMKcHZrdmowSkdyNGJ3TC9iSjNWRHpuME5UbnZWZ3hNWFpHWXB1cWVUTFlOU2UxSHZSL00rSkhCMFdEUUlEQVFBQgpvMU13VVRBZEJnTlZIUTRFRmdRVUJhY0ZrL1crbjc2by9BTjllNHFhekNxaGo5b3dId1lEVlIwakJCZ3dGb0FVCkJhY0ZrL1crbjc2by9BTjllNHFhekNxaGo5b3dEd1lEVlIwVEFRSC9CQVV3QXdFQi96QU5CZ2txaGtpRzl3MEIKQVFzRkFBT0NBUUVBTU80WmRnMENoZ3ZuMTRGeDk5YWx2NkthMTdhZ0hrLzhQenRxQ1RqNGZBa0xsMjdoVUpwSwpDcmJyQnRpMEh5M0lNV2NFL1dCL2N2RlpiY1NNY2I3Yk5qYUFrY1c3YkpDUGgvS1dWa2ptRytYYmxhdWUwa01DCmFQYytmYUFVM0NQWG5XVUlEdmUvRHBXSnIvb054NWN3WFRWK1NiR1FKZE1RYldiTFBBQ1VTaFREMkRPREJOMWMKb01XcFFwbjhTUE1UT3hXTzFlZEJON1hoQjBZQnB6U2s1SGxsakhXU21mM0JnNDk4VGZpdndQdHBFMUVwL0NtTgp0dmhHbE5lY0hVZmRZRnd2WWptVURGNHNHdG95bGIwMTBiWDNXbk5VNW1jVUpWZnJCZkFzLzA3b0xXK25USWY1CnJzSEw0VWUxOEdYRmxncmgyWGZXa1VTVUhMVHdQellPMVE9PQotLS0tLUVORCBDRVJUSUZJQ0FURS0tLS0tCg=="
```
Comment (vinsharm-tibco) [inline samples/app-dev/Connectors/MessagingAndEventPubSub/PulsarSample/PulsarChunkingAndBatching/PulsarBatchingConsumer.json:168]: Before uploading sample please remove the certs from app json.

**Review by vinsharm-tibco — CHANGES_REQUESTED** [review]
Please check the review comment.

**Comment by prapatil-tibco** [comment]
Ok, thanks. I will do the suggested changes.

**Pull request merged**